### PR TITLE
Adding Urban Systems to secondary partners

### LINKED ISSUE #106: Enhancement: Add Urban Systems to secondary partners
### Describe the Current Behavior/Feature:

add urban systems secondary partners

---

### Proposed Implementation:

https://urban.systems/Main_Page to `secondarypartnerslist .js` using logo from their website and description below.

```
"Urban.Systems connects vibrant communities using technology to facilitate civic engagement, deliver services and share resources. Urban’s business model is that of providing “City as a Service” in these following areas (but not limited to):
- Scalable, Open, Distributed, Secure, Transaction Platform
- Branded Community To Guarantee Interoperability Promise
- “Citizen Oriented Identity” To Facilitate Secure Data Transactions
```

### PR DESCRIPTION
**Pull request by andycwilliams**
## This PR:

Resolves #106 

Adds URBAN.SYSTEMS to the list of secondary partners.

---

## Screenshots (if applicable):

---

## Future Steps/PRs Needed to Finish This Work (optional):

Determine whether their summary translates well enough from the original bulleted list to a single paragraph, or if more editing should be done.

This PR is currently slated to merge with main. Can be changed to merge with a new separate branch, if desired.

### CODE REVIEW

**Review by Jared-Krajewski — CHANGES_REQUESTED** [review]
I probably could have specified in the issue a bit beter. Looking to keep the bulleted format used in the issue and on their page https://urban.systems/About.

**Comment by andycwilliams** [comment]
Getting bullet points and line breaks to render is proving unusually onerous. Do you have a preferred method?

**Comment by Jared-Krajewski** [comment]
> Getting bullet points and line breaks to render is proving unusually onerous. Do you have a preferred method?

I would add the lines to be bulleted to a separate area in the object for urban systems.. something like 
```js
bulletedList: [
      'Scalable, Open, Distributed, Secure, Transaction Platform.',
      'Branded Community To Guarantee Interoperability Promise.',
      'Citizen Oriented Identity” To Facilitate Secure Data Transactions.'
    ]
 ```
 
 and then conditionally map it under the typography in the components as <ListItem />
 
 ```js
 {bulletedList &&
          bulletedList.length > 0 &&
          bulletedList.map((listItem, itemIndex) => (
            <ListItem sx={{ display: 'list-item' }} key={itemIndex} variant="body2">
              {listItem}
            </ListItem>
          ))}
```

you will get something like 

Screenshot 2024-02-05 at 7 07 35 PM

since I just did it to make this example I just pushed the code as well. It has some errors in console and needs to be added to smallScreen and formatted for that as well. If you can take it from here 😎 , if not I can polish this up when I have more time.

File: contributing.md [hunk 1]
```diff
@@ -26,11 +26,13 @@ All assets for contributors can be pulled from the [Figma Board](https://www.fig
 
 ## 3. Code of Conduct
 
-By participating in this project, you are expected to uphold our [Code of Conduct](./CODE_OF_CONDUCT.md). Please make sure to read and understand it before contributing. (code of conduct subject to change post Code For America re-branding)
+<!-- TODO: Will this contain its own Code of Conduct file? Or will it simply link to https://github.com/codeforpdx/codeofconduct? -->
```
Comment (Jared-Krajewski) [inline contributing.md]: this should link to GH code of conduct you referenced.

File: contributing.md [hunk 2]
```diff
@@ -26,11 +26,13 @@ All assets for contributors can be pulled from the [Figma Board](https://www.fig
...
+
+By participating in this project, you are expected to uphold our [Code of Conduct](./CODE_OF_CONDUCT.md). Please make sure to read and understand it before contributing.
 
 ## 4. Connect With the Team
 
-- Join [Code for PDX Discord](https://discord.gg/uwqrPpyuap). If you run into any issues join the discord and drop a message in [introductions](https://discord.com/channels/1068260532806766733/1075286322530484256) or [general-chat](https://discord.com/channels/1068260532806766733/1068260535080063028).
+- Join [CODE PDX Discord](https://discord.gg/uwqrPpyuap). If you run into any issues join the discord and drop a message in [introductions](https://discord.com/channels/1068260532806766733/1075286322530484256) or [general-chat](https://discord.com/channels/1068260532806766733/1068260535080063028).
```
Comment (Jared-Krajewski) [inline contributing.md:33]: good catch on all the branding stuff.

**Review by Jared-Krajewski — APPROVED** [review]
🚀  If you want to fix that code of conduct link that would be cool too. If not it can be done separate. otherwise good to go.

**Pull request merged**